### PR TITLE
debugger: vendor debug_toolbar

### PR DESCRIPTION
**Pull request by dcramer**
The new version of django-debug-toolbar is not compatible with Django < 1.8. Additionally we likely want to make significant visual/component changes and make it a core Sentry feature in the short term.

This backs out our custom changes and converts the CSS to less.

#nochanges

### CODE REVIEW

**Comment by dcramer** [comment]
@getsentry/platform

**Comment by dcramer** [comment]
Aside this is version 1.2.3 or whatever is pinned in sentry.io (per the github tag)

File: src/sentry/debug/middleware.py [hunk 1]
```diff
@@ -32,7 +32,6 @@ class DebugMiddleware(object):
     _body_regexp = re.compile(re.escape('</body>'), flags=re.IGNORECASE)
 
     def show_toolbar_for_request(self, request):
-        # TODO(dcramer): support VPN via INTERNAL_IPS + ipaddr maps
```
Comment (mattrobenolt) [inline src/sentry/debug/middleware.py]: 👍

**Review by mattrobenolt — APPROVED** [review]
wcgw

**Pull request merged**